server/index.js

```diff
@@ -1,21 +1,37 @@
 import express from 'express';
 import path from 'path';
+import bodyParser from 'body-parser';
 import webpack from 'webpack';
 import webpackDev from 'webpack-dev-middleware';
+import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
 import webpackConfig from '../webpack.config.babel';
+import schema from './schema';
 
 const app = express();
 const compiler = webpack(webpackConfig);
 
 app.use(webpackDev(compiler, {
-  noInfo: true,
-  publicPath: webpackConfig.output.publicPath,
+    noInfo: true,
+    publicPath: webpackConfig.output.publicPath,
 }));
 
 app.use(express.static(path.resolve(__dirname, '/../public')));
 
+/**
+ Set up GraphiQL and GraphQL
+ **/
+
+app.use('/graphiql', graphiqlExpress({
```

Comment:
There's really no problem with having it in "prod" since it allows you to play with the API. When we implement auth, you won't be able to make requests without a JWT.